### ISSUE TITLE
Rendering broken in Arcade 3.0.0.dev26 and up

### ISSUE THREAD

**Issue by Packman28334** [kfields/arcade-imgui#10]
When attempting to initialize ArcadeRenderer while using Arcade 3.0.0.dev26 and later versions, the class fails to initialize with the following exception:
```
Traceback (most recent call last):
  File "(redacted)/main.py", line 26, in <module>
    window = BasicExample()
  File "(redacted)/main.py", line 12, in __init__
    self.renderer = ArcadeRenderer(self)
  File "(redacted)/arcade-imgui-arcade3/arcade_imgui/__init__.py", line 283, in __init__
    viewport = window.get_viewport()
AttributeError: 'BasicExample' object has no attribute 'get_viewport'. Did you mean: '_viewport'?
```
I'm testing using the example code [here](https://github.com/kfields/arcade-imgui/blob/arcade3/examples/basic.py), running the latest version from the arcade3 branch with the pyproject.toml modified to allow installation on later dev builds of Arcade 3.

I did attempt to change the usages of `window.get_viewport()` to both `window.ctx.viewport` and `window._viewport` (which seemed to do same thing as the original function, however I don't have any experience with low-level graphics rendering, so I'm not sure) which allowed the script to launch, but no actual imgui windows were displayed.

Tested on Arcade 3.0.0.dev24, 25, 26, and 32. The latter two didn't work while the former worked flawlessly.
Linux Mint Cinnamon 21.3, Python 3.10.12.

**Comment by Packman28334** [comment]
Just found this [Gist](https://gist.github.com/eruvanos/dcef0132c0f2b535fdd4382d9137f74a) which I've tested and confirmed that it works on dev32. Maybe it's worth taking a look at?

**Comment by kfields** [comment]
Fixed: [update to arcade 3.0.0.dev32](https://github.com/kfields/arcade-imgui/commit/b4541d839429c0e744ca8f88720d31cb24d6470a)
They renamed `get_viewport` to `get_framebuffer_size`.
I also updated the demos.
